### DISCUSSION TITLE
Article contents field for creation and update have different names, articletext vs fulltext

### DISCUSSION THREAD

**Discussion by Taz-1999** [alexandreelise/j4x-api-collection#43] [General]
Hi

Just discovered this API collection and want to start with many thanks to Alexandreelise for making them.

I ran into a problem with updating already created articles suing the API calls. 

It seems the full article text field changes name in between create and update? 

This example is used in the collection and help and Joomla! API specs to CREATE an article.

```
{
    "alias": "my-article",
    "articletext": "My text",
    "catid": 64,
    "language": "*",
    "metadesc": "",
    "metakey": "",
    "title": "Here's an article"
}
```

https://docs.joomla.org/J4.x:Joomla_Core_APIs#Create_Article

And the cURL url is POST {{base_url}}/{{base_path}}/content/articles etc. 

So the _articletext_ is what the content of the article is. 

But there is no example for UPDATE that contains the _articletext_, so I used the same fieldname _articletext_. And after many experiments using other names instead of _articletext_ (text, article, content attributes.text and others), since _articlectext_ didn't work, I discovered that the correct field name for the articletext is now... **_fulltext_**. 

```
{
    "fulltext": "<p>Testing testing testing</p>"
}
```

And the cURL url is PATCH {{base_url}}/{{base_path}}/content/articles/{article_id}

Also, ypu need to specify _introtext_ separately, when updating.

I have no idea why that is, or where I would find that documented. That it's _articletext_ in one place and _fulltext_ in another. Why?!

**Suggestion**: Add _fulltext_ + _introtext_ as field names in the API-collection documentation and example for UPDATE. 

Thanks again for making this collection. Superuseful!

/Ola

**Comment by alexandreelise** [comment]
Hi @Taz-1999 Thanks for sharing your thoughts on this postman collection in discussions. It's the right place to ask those kind of question and discuss about a particular feature or missing documentation. I'm not working with the core team of joomla and don't pretend to know all the answers about your concerns maybe @brianteeman or @zero-24 might have more info on this one. From my side, I'll check to see what's happening for me and give you some feedback when I have some time. Thanks again for sharing your insights and have a great day.
